src/jobflow_remote/jobs/data.py

```diff
@@ -584,7 +585,9 @@ def from_query_dict(cls, d) -> "FlowInfo":
     def ids_mapping(self) -> dict[str, dict[int, str]]:
         d: dict = defaultdict(dict)
 
-        for db_id, job_id, index in zip(self.db_ids, self.job_ids, self.job_indexes):
+        for db_id, job_id, index in zip(
+            self.db_ids, self.job_ids, self.job_indexes, strict=False
```

Comment:
Shouldn't it be strict=True here as we want to have db ids, job ids and job indices to be of the same size I guess.